practice/src/scripts/constants/labels.ts

```diff
@@ -0,0 +1,42 @@
+export enum PRODUCT_TYPE {
+  ALL_CATEGORY = 'All category',
+  SKIN_CARE = 'Skin care',
+  FACE_CARE = 'Face care',
+  LIPS_CARE = 'Lips care',
+}
+
+export enum PRODUCT_STATUS {
+  ALL_STATUS = 'All status',
+  BEST_SELLER = 'Best-seller',
+  POOR_SELLER = 'Poor seller',
+  ON_SALE = 'On sale',
+  NEW_ARRIVAL = 'New arrival',
+  LOW_STOCK = 'Low stock',
+}
+
+export const PRODUCT_LABEL = {
```

Comment:
Why do we have to use object-key-object here when you just using label?
-  If you only need the label, we can use just string array
```
export const PRODUCT_LABELS = ['Product name', 'Category', ...]
```

- If you need to use other fields later, we can use something like:
```
export const PRODUCT_HEADER_COLUMNS = [
  {
    key: 'name',
    label: 'Product name',
    isSorting: true
  }
...
]
```